src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzerTests.GuardedCallsTests.cs

```diff
@@ -120,19 +120,17 @@ void M1()
         }
         else
         {
-            [|WindowsOnlyMethod<BrowserOnlyType>()|];  // should flag for WindowsOnlyMethod method and BrowserOnlyType parameter
-            [|GenericMethod<WindowsOnlyType, BrowserOnlyType>()|]; // should flag for WindowsOnlyType and BrowserOnlyType parameters
+            {|#0:WindowsOnlyMethod<BrowserOnlyType>()|};  // should flag for WindowsOnlyMethod method and BrowserOnlyType parameter
+            {|#1:GenericMethod<WindowsOnlyType, BrowserOnlyType>()|}; // should flag for WindowsOnlyType and BrowserOnlyType parameters
         }
     }
 }
 ";
             await VerifyAnalyzerAsyncCs(csSource, s_msBuildPlatforms,
-#pragma warning disable RS0030 // Do not used banned APIs
-                VerifyCS.Diagnostic(PlatformCompatibilityAnalyzer.OnlySupportedCsAllPlatforms).WithLocation(24, 13).WithArguments("BrowserOnlyType", "'browser'"),
-#pragma warning restore RS0030 // Do not used banned APIs
-#pragma warning disable RS0030 // Do not used banned APIs
-                VerifyCS.Diagnostic(PlatformCompatibilityAnalyzer.OnlySupportedCsAllPlatforms).WithLocation(25, 13).WithArguments("WindowsOnlyType", "'windows'"));
-#pragma warning restore RS0030 // Do not used banned APIs
```

Comment:
The 'browser' diagnostic was only showing up for the first invocation, and the 'windows' diagnostic was only showing up for the second invocation?

The fix makes sense (both diagnostics should show up for both invocations) but I am not understanding why is it related to the problem this PR is trying to address.

Reply:
> The fix makes sense (both diagnostics should show up for both invocations) but I am not understanding why is it related to the problem this PR is trying to address.

This doesn't relate to the problem, as you can see `WithLocation(int, int)` is banned, previously i have had updated all such usages with `WithLocation(int)` but somehow this one left, i guess it is caused by many different branches merge conflicts. Anyway, this change just to replace the deprecated API usage and remove the suppression, no any functional/test change here